closed_issues.rb

```diff
@@ -7,8 +7,9 @@
 
 File.open('closed_issues_master_repo.csv', 'w') do |f|
     f.puts "Milestone Statistics for: #{prs.first.milestone.title}"
-    f.puts "NUMBER,TITLE,AUTHOR,ASSIGNEE,LABELS"
+    f.puts "NUMBER,TITLE,AUTHOR,ASSIGNEE,LABELS,CHANGELOGTEXT"
     prs.each do |i|
-    	f.puts "#{i.number},#{i.title},#{i.user.login},#{i.assignee && i.assignee.login},#{i.labels.collect(&:name).join(" ")}"
+    	i.changelog = "#{i.title} #[#{i.number}](https://github.com/ManageIQ/manageiq/pull/#{i.number})"
```

Comment:
so, I made things harder on myself.

👍 

at least I got the concatenation right!